scripts.js

```diff
@@ -131,144 +124,85 @@ function getRandomIndex(array) {
+    savedPalette.classList.add("saved-palette");
+    savedPaletteSection.appendChild(savedPalette);
     var colorBox1 = document.createElement("div");
     colorBox1.classList.add("saved-color");
-    savedPalette.appendChild(colorBox1)
-    colorBox1.style.backgroundColor = `${savedPalettes[i].firstColor.hexCode}`
+    savedPalette.appendChild(colorBox1);
+    colorBox1.style.backgroundColor = `${savedPalettes[i].firstColor.hexCode}`;
     var colorBox2 = document.createElement("div");
     colorBox2.classList.add("saved-color");
-    savedPalette.appendChild(colorBox2)
-    colorBox2.style.backgroundColor = `${savedPalettes[i].secondColor.hexCode}`
+    savedPalette.appendChild(colorBox2);
+    colorBox2.style.backgroundColor = `${savedPalettes[i].secondColor.hexCode}`;
     var colorBox3 = document.createElement("div");
     colorBox3.classList.add("saved-color");
-    savedPalette.appendChild(colorBox3)
-    colorBox3.style.backgroundColor = `${savedPalettes[i].thirdColor.hexCode}`
+    savedPalette.appendChild(colorBox3);
+    colorBox3.style.backgroundColor = `${savedPalettes[i].thirdColor.hexCode}`;
     var colorBox4 = document.createElement("div");
     colorBox4.classList.add("saved-color");
-    savedPalette.appendChild(colorBox4)
-    colorBox4.style.backgroundColor = `${savedPalettes[i].fourthColor.hexCode}`
+    savedPalette.appendChild(colorBox4);
+    colorBox4.style.backgroundColor = `${savedPalettes[i].fourthColor.hexCode}`;
     var colorBox5 = document.createElement("div");
     colorBox5.classList.add("saved-color");
-    savedPalette.appendChild(colorBox5)
-    colorBox5.style.backgroundColor = `${savedPalettes[i].fifthColor.hexCode}`
+    savedPalette.appendChild(colorBox5);
+    colorBox5.style.backgroundColor = `${savedPalettes[i].fifthColor.hexCode}`;
     var trash = document.createElement("div");
     trash.classList.add("trash-icon");
-    savedPalette.appendChild(trash)
+    trash.setAttribute("id", `${savedPalettes[i].id}`);
+    savedPalette.appendChild(trash);
     }
-
-}
-
-// data-hexcode="${savedPalettes[i].firstColor.hexCode}"
-
-function deletePalette(event) {
-   if (event.target.className === '.trash') {
-
-   }
-}
-
-// function generateInitialColors() {
-//   color1 = new Color(generateHexCode())
-//   colorOneDiv.style.backgroundColor = color1.hexCode
-//   hexCodeOne.innerText = color1.hexCode
-//   color2 = new Color(generateHexCode())
-//   colorTwoDiv.style.backgroundColor = color2.hexCode
-//   hexCodeTwo.innerText = color2.hexCode
-//   color3 = new Color(generateHexCode())
-//   colorThreeDiv.style.backgroundColor = color3.hexCode
-//   hexCodeThree.innerText = color3.hexCode
-//   color4 = new Color(generateHexCode())
-//   colorFourDiv.style.backgroundColor = color4.hexCode
-//   hexCodeFour.innerText = color4.hexCode
-//   color5 = new Color(generateHexCode())
-//   colorFiveDiv.style.backgroundColor = color5.hexCode
-//   hexCodeFive.innerText = color5.hexCode
-// }
+};
```

Comment:
Something that could be useful here is the `template` html element. The idea is that you can build a generic templated element in your html and then call it without having to create all the elements on the dom. Certainly not something that needs to be refactored right here in this instance, just another thing to add onto the limitless pile of information in front of you 🙃